rust/worker/src/processor.rs

```diff
@@ -0,0 +1,100 @@
+use crate::worker::SerializedBatchDigestMessage;
+use config::WorkerId;
+use crypto::Digest;
+use ed25519_dalek::Digest as _;
+use ed25519_dalek::Sha512;
+//use primary::WorkerPrimaryMessage;
+use std::convert::TryInto;
+use store::Store;
+use tokio::sync::mpsc::{Receiver, Sender};
+
+// TODO: Uncomment when we cleanup the primary.
+//#[cfg(test)]
+//#[path = "tests/processor_tests.rs"]
+//pub mod processor_tests;
+
+/// Indicates a serialized `WorkerMessage::Batch` message.
+pub type SerializedBatchMessage = Vec<u8>;
+
+/// Hashes and stores batches, it then outputs the batch's digest.
+pub struct Processor;
+
+/*
+impl Processor {
```

Comment:
delete?

Reply:
It is deleted in the next PR. The commented code is the "real one", the rest is just a glue for the old-primary.